### DISCUSSION TITLE
Umbraco Should have more explicit notification system

### DISCUSSION THREAD

**Discussion by bielu** [umbraco/Umbraco-CMS#13444] [Features and ideas]
After discussion in this ticket:
https://github.com/umbraco/Umbraco-CMS/issues/13339#event-7814272212
I think it should be a wider discussion about Umbraco notifications, as right now there is a chance some events are used by multiple services and might cause random triggers of events.

**Comment by lassefredslund** [comment]
Hi @bielu. Thank you for the suggestion. I'll bring this up with the team when I get the chance. I could fear this being a big task and potentially breaking, but let me come back to you when I have some thoughts from the team.

**Comment by bielu** [comment]
@lassefredslund it is a big task and for sure breaking, but it is something that should be address to avoid a bad developer experience :)
